tikv-client/src/main/java/com/pingcap/tikv/AbstractGRPCClient.java

```diff
@@ -34,16 +35,12 @@
         BlockingStubT extends AbstractStub<BlockingStubT>, StubT extends AbstractStub<StubT>>
     implements AutoCloseable {
   protected final Logger logger = Logger.getLogger(this.getClass());
-  protected TiSession session;
-  protected TiConfiguration conf;
+  protected final TiConfiguration conf;
```

Comment:
can you add some comments why this line should be changed?

Reply:
conf will not change after initialization, it's better to use final.